src/connectors/infinite-hits/__tests__/connectInfiniteHits-test.ts

```diff
@@ -805,10 +809,12 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/infinite-hi
         }
       );
 
-      expect(actual).toEqual({});
+      expect(actual).toEqual({
```

Comment:
If `uiState` is empty, then `getWidgetSearchParameters` doesn't get the `page` and won't be able to refine the search correctly.